### ISSUE TITLE
Audacious plugin build fails: expected unqualified-id before ‘)’ token

### ISSUE THREAD

**Issue by ToadKing** [losnoco/vgmstream#491]
```
g++ -DHAVE_CONFIG_H -I. -D_FORTIFY_SOURCE=2 -DVERSION=\"r1050-2553-gee3e1700\" -Wall -std=c++11 -fpermissive -I.. -I.. -I../ext_includes/ -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/gtk-2.0 -I/usr/lib/gtk-2.0/include -I/usr/include/pango-1.0 -I/usr/lib/libffi-3.2.1/include -I/usr/include/harfbuzz -I/usr/include/fribidi -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/atk-1.0 -pthread -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -MT plugin.lo -MD -MP -MF .deps/plugin.Tpo -c plugin.cc  -fPIC -DPIC -o .libs/plugin.o
In file included from plugin.cc:21:
plugin.h:31:13: error: expected unqualified-id before ‘)’ token
   31 |             ) {}
      |             ^
```

Operating system: Linux linsux 5.3.7-arch1-1-ARCH <span>#</span>1 SMP PREEMPT Fri Oct 18 00:17:03 UTC 2019 x86_64 GNU/Linux

GCC version: 9.2.0

**Comment by kode54** [comment]
This should be fixed by the latest commit, 9bfc3c0.

**Comment by ToadKing** [comment]
Confirmed fixed, thanks!
